server/src/main/resources/mail/candidate-duolingo-coupon.html

```diff
@@ -0,0 +1,108 @@
+<div>
+    <h4>Steps to take the Duolingo English Test (DET)</h4>
+    <p>Dear candidate,</p>
+    <b>
+        We hope you are doing well. You are receiving this email because we need you to take the Duolingo English Test
+        (DET).
+    </b>
```

Comment:
This is all good text. But now it will be displayed in the new DuoLingo tab of the candidate portal.

The notification email can be much simpler - just informing them that they can take the DET - directing them to login to their candidate portal for more details.